extensions/vertx-http/deployment/src/main/java/io/quarkus/devui/deployment/menu/ConfigurationProcessor.java

```diff
@@ -300,7 +301,7 @@ private List<ConfigDescription> calculate(List<ConfigDescription> cd, Set<String
     private ConfigValue getConfigValue(SmallRyeConfig config, String name) {
         try {
             return config.getConfigValue(name);
-        } catch (java.util.NoSuchElementException nse) {
+        } catch (NullPointerException | NoSuchElementException nse) {
```

Comment:
Hmmm, is it the right fix? It's normal that we don't have a context set in this particular case?

/cc @radcortez

Reply:
Please check: https://github.com/quarkusio/quarkus/issues/33884#issuecomment-1584417651